android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
```

Comment:
I recommend that you use the "all" distribution type (not "bin") so that it will include docs.
```sh
./gradlew --gradle-version 7.4.2 --distribution-type all
```

Reply:
Thanks for the info! I don't have that flag `--distribution-type` :thinking:  But I've changed the file to all

Reply:
Sorry. I wrote it wrong. Need to add "wrapper" command to it first.
```sh
./gradlew wrapper --gradle-version 7.4.2 --distribution-type all
```

Reply:
ah ok :+1: no changes after that, but good to know. I only used the first part of it from the gradle docs